pkg/proto/sqlflow.proto

```diff
@@ -34,43 +34,38 @@ service SQLFlow {
     // Fetch fetches the SQLFlow job phase and logs in a polling manner. A corresponding
     // client can be implemented as
     //
-    // job = c.run(request).job
-    // token = fetch_token(job=job)
-    // while True:
-    //     r = c.fetch(token)
-    //     print(r.logs.content)
-    //     if r.phase == SUCCEEDED/FAILED/ERROR and r.logs.no_more_log:
-    //         break
-    //     token = r.token
-    rpc Fetch (FetchToken) returns (FetchResponse);
+    // wfJob := Submit(argoYAML)
+    // req := newFetchRequest(wfJob)
```

Comment:
From https://github.com/sql-machine-learning/sqlflow/pull/1488/files#diff-b03e31960a96575596cdfc09b491fefaR25, it seems that function `newFetchRequest` has 3 parameters.  However, in the pseudo-code here, it has only one.  How to set the rest two?

Reply:
The rest two would be empty string, maybe the following code is clearer:

``` go
req := &FetchRequest {
  Job: &pb.Job{
    Id: wfJob,
   },
}
```